compass/ocean/inactive_top_cells.py

```diff
@@ -0,0 +1,28 @@
+import os
+import xarray
+
+from mpas_tools.io import write_netcdf
+
+
+def remove_inactive_top_cells_output(in_filename, inactive_top_cells=1):
+    """
+    Remove inactive top cells from the output netCDF file
+
+    Parameters
+    ----------
+    in_filename : str
+        Filename for the netCDF file to be cropped
+
+    inactive_top_cells : str, optional
+        The number of inactive top cell layers. It should be equal to
+        ``config.inactive_top_cells``.
```

Comment:
I take back what I said about passing the `config`.  That won't work because you want to also call this function from a `ForwardStep`, where the config option is not available (or, really, not useful because it doesn't necessarily have the same value as in the `Init` test case).

My suggestion here would be to not pass the `inactive_top_cells` at all bur rather to read `minLevelCell` from the file, raise an error if it's not a constant value, an use that constant value as the offset.  This should be a good solution in the user has changes `inactiv_top_cells` in the config file in `init`.